generators/markup/files/render.js

```diff
@@ -0,0 +1,47 @@
+
+// React.
+import { createElement } from 'react';
+import { renderToStaticMarkup, renderToString } from 'react-dom/server';
+
+// Prevent XSS.
+import escape from 'htmlescape';
+
+// Generic page layout.
+import Page from './page';
+import Root from '../';
+
+function body(root, { scripts = [] }) {
+  // TODO: This is where any flux magic would go.
+  return new Promise((resolve) => {
```

Comment:
What default flux magic goes here? `redux` and friends have to load the correct state basically.